template.yaml

```diff
@@ -3283,7 +3284,7 @@ Resources:
         - !Ref IpvTokenSigningKmsAccessPolicy
         - !Ref SpotRequestQueueWritePolicy
         - !Ref AccountInterventionsServiceUriSecretAccessPolicy
-        - !Ref OrchSessionTableReadAccessPolicy
+        - !Ref OrchSessionTableReadAndWriteAccessPolicy
```

Comment:
It's required because in IPVCallbackHandler, there's a possibility that `handleAccountInterventionLogout` is hit, which is a `LogoutService` method that deletes the session.

Agree that separate policies is a good idea